### PR TITLE
Deletegame function added with foreign keys

### PR DESCRIPTION
**Pull request by monodrail**

### CODE REVIEW

File: VS_Project/Pokerapp/Pokerapp/Queries/InitiateDatabase.sql [hunk 1]
```diff
@@ -1,14 +1,7 @@
...
-    [Room_Code] INT NOT NULL, 
-    [Tbl_nr] INT NULL, 
-    [Rebuy] INT NULL,
-    PRIMARY KEY ([Name],[Room_Code]), 
-)
-
+DROP TABLE Blinds;
 DROP TABLE Room;
+
 CREATE TABLE [dbo].[Room] (
```
Comment (jorritschippers) [inline VS_Project/Pokerapp/Pokerapp/Queries/InitiateDatabase.sql:5]: [dbo].[Room]  moet Room worden

File: VS_Project/Pokerapp/Pokerapp/Queries/InitiateDatabase.sql [hunk 2]
```diff
@@ -35,11 +28,27 @@ CREATE TABLE [dbo].[Room] (
...
+    [Tbl_nr] INT NULL, 
+    [Rebuy] INT NULL,
+    PRIMARY KEY CLUSTERED ([Name] ASC, [Room_Code] ASC),
+    CONSTRAINT [FK_Player_Room_Code] FOREIGN KEY ([Room_Code])
+		REFERENCES [dbo].[Room] ([Room_Code])
+		ON DELETE CASCADE
+		ON UPDATE CASCADE
+)
+
 CREATE TABLE [dbo].[Blinds] (
```
Comment (jorritschippers) [inline VS_Project/Pokerapp/Pokerapp/Queries/InitiateDatabase.sql:44]: Same

**Review by jorritschippers — CHANGES_REQUESTED** [review]
Kijk comments

File: VS_Project/Pokerapp/Pokerapp/Queries/InitiateDatabase.sql (hunk 1 above)
Comment (monodrail) [reply VS_Project/Pokerapp/Pokerapp/Queries/InitiateDatabase.sql:5]: niet zo zeuren jorrita

**Review by jorritschippers — APPROVED** [review]

**Review by NKN90 — APPROVED** [review]
Nice

**Pull request merged**